common/changes/@itwin/presentation-frontend/presentation-imodelkey-selection_2024-12-17-12-36.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-frontend",
+      "comment": "Expose `SelectionManager.selectionStorage` as a readonly member. Consumers gives consumers a chance to start using selection storage directly instead of through soon-to-be-deprecated `SelectionManager`.",
```

I guess it should be:
```suggestion
      "comment": "Expose `SelectionManager.selectionStorage` as a readonly member. This gives consumers a chance to start using selection storage directly instead of through soon-to-be-deprecated `SelectionManager`.",
```